Practice/assets/javascript/smoothscroll.js

```diff
@@ -0,0 +1,46 @@
+/* 
+The code to activate smooth scroll.
+https://github.com/cferdinandi/smooth-scroll
+
+*/

```

Comment:
The smooth scroll library is not required if you have put in the css property which you have at the top of your page